### PR TITLE
Steph branch

### PR DESCRIPTION
**Pull request by whatawhat**

### CODE REVIEW

File: client/src/pages/Account.js [hunk 1]
```diff
@@ -51,18 +42,16 @@ const Account = () => {
   if (!profile?.name) {
```
Comment (Captain63) [inline client/src/pages/Account.js:42]: We don't need this since they can't reach this page without being signed in

**Review by Captain63 — CHANGES_REQUESTED** [review]
See comment. Also make sure you have pulled the latest version of main and merged. I couldn't recreate the previous error after pulling main, so that should fix the issue

**Comment by whatawhat** [comment]
I removed that part out of my code. I also made sure I pulled and merged the latest version.

**Review by Captain63 — APPROVED** [review]
Looking good.

**Pull request merged**